ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -104,9 +104,9 @@ conformance_test(
         "@jdk11_nix//:bin/java",
     ],
     lf_versions = [
-        "stable",
-        # FIXME: add latest once canton-test-runner supports LF 1.13
-        #  "latest",
+        # FIXME: replace "legacy" by "stable", "latest" once canton-test-runner supports LF 1.13
```

Comment:
So this should no longer be necessary with 1e009b4ac2c24d118351f1af1e9689972def1dea